### PR TITLE
chore: Use Node 16 in CI

### PR DESCRIPTION
**Pull request by nschonni**
Now that it is the LTS branch

### CODE REVIEW

**Review by trivikr — APPROVED** [review]

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -27,7 +24,7 @@ jobs:
       - name: Set up Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: "${{ env.NODE }}"
+          node-version: 16
```
Comment (Trott) [inline .github/workflows/test.yml]: For what it's worth, I believe this should always install the latest LTS release:

```suggestion
          node-version: lts/*
```

**Review by Trott — APPROVED** [review]
LGTM with or without my suggestion (since I only *think* my suggestion will work but I'm not sure)

**Comment by humble-barnacle001** [comment]
Dependency installation with node 16 gives the depreciation warnings:
```
npm WARN deprecated sprintf@0.1.5: The sprintf package is deprecated in favor of sprintf-js.
npm WARN deprecated fs-tools@0.5.0: Use shelljs, glob, mkdirp, rimraf, or other alternatives
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
```

**Review by DerekNonGeneric — APPROVED** [review]
According to the docs, that does appear to be a valid version range, which allows the nvm LTS syntax: `lts/erbium`, `lts/fermium`, `lts/*`. Guess that would be the Bash version of the command (they do not specify).

Refs: https://github.com/actions/setup-node#usage

**Pull request merged**